### PR TITLE
Feature/running with many reporters

### PR DESCRIPTION
**Pull request by jgebal**

### CODE REVIEW

**Comment by jgebal** [comment]
Hi @pazus. I've udated the PR with changes and addressed your comment.

**Comment by jgebal** [comment]
I'm just thinking if the ut_run_sqlplus.sql should be in `/sources` or do we need a new directory called `/client` to store examples of client side code used for invoking UTPLSQL with all it's power and reporting capabilities.

**Comment by Pazus** [comment]
How about placing it in the root?

File: source/core/types/ut_output_stream.tpb [hunk 1]
```diff
@@ -23,3 +23,8 @@ create or replace type body ut_output_stream as
 
 end;
 /
+--we need to use plsql optimize level=1 to prevent
```
Comment (Pazus) [inline source/core/types/ut_output_stream.tpb]: How about changing the " for loop" to open-loop-fetch-close? I thought optimization leaves it as is. This way altering plsql_optimizate_level is not needed (but should be rechecked)

Comment (jgebal) [inline source/core/types/ut_output_stream.tpb]: @pazus Can you do that change?

**Pull request merged**